topics/tutorials/code_documentation.md

```diff
@@ -3,13 +3,31 @@
 <!-- Copyright 2000-2022 JetBrains s.r.o. and other contributors. Use of this source code is governed by the Apache 2.0 license that can be found in the LICENSE file. -->
 
 [Quick Documentation](https://www.jetbrains.com/help/idea/viewing-reference-information.html#inline-quick-documentation)
-helps users by showing documentation for symbols like method calls inside the editor.
+helps users by showing documentation, e.g., for classes, functions, or methods inside the editor.
 Plugin authors implement
 [`DocumentationProvider`](upsource:///platform/analysis-api/src/com/intellij/lang/documentation/DocumentationProvider.java)
 to show documentation for particular Psi elements.
 
 `DocumentationProvider` can be used for existing languages as well as for a custom language implementation.
 For an existing language, the implementation of `DocumentationProvider` is registered as `com.intellij.documentationProvider` extension point (EP),
 while custom languages should register it as `com.intellij.lang.documentationProvider` EP.
-For details on how to implement a `DocumentationProvider`, please refer to the [Custom Language Support](documentation_provider.md)
+The difference between both is that the latter requires the `language` attribute when registering the EP in the `plugin.xml` and is only called
```

Comment:
use `<path>` for filename

Reply:
Oh boy, when will I finally remember that one? Every. Single. Time.